index.html

```diff
@@ -5931,7 +5931,31 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content represents additional information or parenthetical context to the primary content it supplements. A <code>note</code> is content provided by the author of the page or document, it is not to be used for providing reactions or suggestions. For these purposes, please review <rref>comment</rref> and <rref>suggestion</rref>.</p>
```

Comment:
```suggestion
				<p>A <rref>section</rref> whose content represents additional information or parenthetical context to the primary content it supplements.</p>
				<p>A <code>note</code> is content provided by the author of the page or document, it is not to be used for providing reactions or suggestions. For these purposes, please review <rref>comment</rref> and <rref>suggestion</rref>.</p>
```

Reply:
Keeping the first paragraph short, since it's used in the short role description list here: https://w3c.github.io/aria/#role_definitions